### ISSUE TITLE
Button drawing offset issue

### ISSUE THREAD

**Issue by cfecteau** [dkazmer/sGlide#5]
I have two divs inline. The first div has a a series of thumbnails in it. The second div contains two main vertically stacked divs. The top div contains the picture and the bottom div contains your slider. The issue is that when I drag the slider button the button is immediately offset to the right by the width of the first div containing the thumbnails. If I hide the first div then the control works normally. 

Thanks,
Chuck

**Comment by dkazmer** [comment]
Can you put it in a jsbin so I can have a look?

Sent from my Windows Phone
________________________________

**Comment by cfecteau** [comment]
Hey Daniel,

I'll zip it and send it to you when I can. I'm in the middle of re-working it by putting the div for the slider inside another at the bottom of the form and using some javascript to move it into place with absolute positioning. It appears to be a simple calculation issue where you are getting the screen coords of the mouse pointer and translating during the button draw to the coords of the div window which are offset because of the div to the left. 

I'm working around it because I really like the look of your slider which will look sharp on my viewer.

Chuck


--------------------------------------------
On Fri, 1/16/15, Daniel Kazmer <notifications@github.com> wrote:

 Subject: Re: [sGlide] Button drawing offset issue (#5)
 To: "dkazmer/sGlide" <sGlide@noreply.github.com>
 Cc: "cfecteau" <Chuck_Fecteau@yahoo.com>
 Date: Friday, January 16, 2015, 11:45 AM
 
 Can you put it in a jsbin so I can
 have a look?
 
 
 
 
 
 Sent from my Windows Phone
 
 
 ________________________________

**Comment by dkazmer** [comment]
Hi Chuck,
I've tried to replicate your build here: http://jsbin.com/wabiyoyelu
So far it works as expected...
I'm about the release a patch, and I'd like to address your issue as well, if I can.

**Comment by cfecteau** [comment]
Sorry Daniel, 

I just ended up ripping it out and using a regular html5 slider (input range). I was spending way
too much time trying to get it into place and sized up right. Also had an issue with the +- buttons
not working when the slider was at it's extremes. I wish I had taken a snapshot at the time I was 
having the issue to send you but I'm so far past it and behind schedule I don't have time to mess
with it.

Sorry,
Chuck



--------------------------------------------
On Sat, 1/24/15, Daniel Kazmer <notifications@github.com> wrote:

 Subject: Re: [sGlide] Button drawing offset issue (#5)
 To: "dkazmer/sGlide" <sGlide@noreply.github.com>
 Cc: "cfecteau" <Chuck_Fecteau@yahoo.com>
 Date: Saturday, January 24, 2015, 2:40 PM
 
 Hi Chuck,
 
 I've tried to replicate your build here: http://jsbin.com/wabiyoyelu
 
 So far it works as expected...
 
 I'm about the release a patch, and I'd like to
 address your issue as well, if I can.
 
 —
 Reply to this email directly or view
 it on GitHub.
